packages/popover/src/popover.scss

```diff
@@ -24,73 +25,47 @@ $size-arrow: 8px;
           display: none;
         }
       }
+    }
 
-      .af-popover__arrow {
-        width: 0;
-        height: 0;
-        border-style: solid;
-        position: absolute;
-      }
-
-      &[data-placement^='top'] {
-        margin-bottom: $size-arrow;
-
-        .af-popover__arrow {
-          border-width: $size-arrow $size-arrow 0 $size-arrow;
-          border-color: $color-azur transparent transparent transparent;
-          bottom: -$size-arrow;
-          left: calc(50% - #{$size-arrow});
-          margin-top: 0;
-          margin-bottom: 0;
+    &--small {
+      .af-btn {
+        &--circle {
+          width: 2rem;
+          height: 2rem;
         }
       }
+    }
 
-      &[data-placement^='bottom'] {
-        margin-top: $size-arrow;
-
-        .af-popover__arrow {
-          border-width: 0 $size-arrow $size-arrow $size-arrow;
-          border-color: transparent transparent $color-azur transparent;
-          top: -$size-arrow;
-          left: calc(50% - #{$size-arrow});
-          margin-top: 0;
-          margin-bottom: 0;
-        }
+    .af-popover__arrow {
+      width: $size-arrow;
+      height: $size-arrow;
+      position: absolute;
+      z-index: -1;
+
+      &::before {
```

Comment:
The arrow is now in a `::before` pseudo-element so I rewrote it so it's now easier to override